### ISSUE TITLE
Upgrade to Python 3?

### ISSUE THREAD

**Issue by gwaygenomics** [greenelab/staNMF#9]
Has there been any thought about implementing staNMF in python 3? I am assuming we are restricted to 2.7 because of `SPAMS`?

**Comment by amyecampbell** [comment]
Yeah, that's one thing I've considered. It wouldn't be hard to implement it in python3 except for SPAMS. I'm pretty sure SPAMS is only available for python 2.7. There's also the possibility of trying to implement it in python 3 using a different NMF package, and comparing results
